### DISCUSSION TITLE
Tonal integration or adaptation

### DISCUSSION THREAD

**Discussion by mattaeus-isenberg** [philosowaffle/peloton-to-garmin#420] [General]
How challenging would it be to integrate or adapt this project for Tonal? Do you think its even possible @philosowaffle ?

**Comment by philosowaffle** [comment]
TLDR; Looks like Tonal has made this challenging to near impossible.  

I couldn't find a place to sign in or create an account on the web (the easiest place for me to reverse engineer whats happening). I went ahead and downloaded the app, but looks like there is no way to take a class without owning a Tonal, so hit a dead end there.

While poking around in the app I noticed that they do have native support for syncing to Strava.  Strava is more friendly to 3rd party tools.

There is an existing project [Tapiriik](https://github.com/cpfair/tapiriik) that provides more generic sync functionality between services like Strava and Garmin.  So you might try doing Tonal -> Strava -> Garmin
